.github/pull_request_template.md

```diff
@@ -12,10 +12,10 @@ _Explain what you did test and what you didn't, and why_
 
 ### Screenshots
 
-| Before | After |
-| ------ | ----- |
-| gif/png _before_ | gif/png _after_ |
+ Before | After
```

Comment:
I took the opportunity to clean this up a bit as I needed file changes to trigger the CI for 💃